### PR TITLE
General benchmarks that provide results for all SDK's

### PR DESCRIPTION
**Pull request by medikoo**
At this point just Node.js and Python.

It'll be good to also cover here Go (cc @czubocha )

### CODE REVIEW

File: node/test/benchmark/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+- `SLS_ORG_ID`- Console organization id (can by dummy, as it'll be part of generated trace which otherwise is not send to console servers)
+- `TEST_UID` - (optional) common name token to be used as part of generated resource names. All generated resource names will be prefixed with `test-oext-<test-uid>`. If not provided, one is generated on basis of [local machine id](https://www.npmjs.com/package/node-machine-id). Note: Script ensures that all generated resources are removed after benchmark is done
+- `LOG_LEVEL` - (optional) For more verbose output `LOG_LEVEL=info` can be used
+
+### 4. Run
+
+Benchmark for all configured use cases and benchmark variants can be run as:
+
+```bash
+./node/test/scripts/benchmark.js
```
Comment (czubocha) [inline node/test/benchmark/README.md]:
```suggestion
./node/test/scripts/benchmark/index.js
```

File: node/test/benchmark/README.md [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+```
+
+Generated benchmark results are output to the console in CSV format. To store them to the file output can be piped as:
+
+```bash
+./test/scripts/benchmark.js > benchmark.csv
```
Comment (czubocha) [inline node/test/benchmark/README.md]:
```suggestion
./node/test/scripts/benchmark/index.js > benchmark.csv
```

File: node/test/benchmark/README.md [hunk 3]
```diff
@@ -0,0 +1,139 @@
+# Benchmarks
+
+## Tested use cases
+
+### Node.js success function ([`node-success`](../../packages/aws-lambda-sdk/test/fixtures/lambdas/callback.js))
+
+No-op Node.js syccess function. Handler takes callback and immediately invokes it with dummy payload
```
Comment (Danwakeem) [inline node/test/benchmark/README.md]:
```suggestion
No-op Node.js success function. Handler takes callback and immediately invokes it with dummy payload
```

**Review by Danwakeem — COMMENTED** [review]
Looking good I think I just saw one typo in there 👍

File: node/test/benchmark/README.md (hunk 3 above)
Comment (medikoo) [reply node/test/benchmark/README.md]: Thanks, updated

File: node/test/benchmark/README.md (hunk 1 above)
Comment (medikoo) [reply node/test/benchmark/README.md]: Thanks @czubocha ! Updated

**Review by Danwakeem — APPROVED** [review]
Looking good 👍

**Pull request merged**